packages/notification-system/src/components/NotificationsDashboard/NotificationsDashboard.module.scss

```diff
@@ -0,0 +1,18 @@
+.dashboard {
+	background-color: #fff;
```

Comment:
Is there any convention related with the colors at cohort level?

Reply:
I didn't find related threads yet